apps/console/src/features/applications/components/forms/inbound-saml-form.tsx

```diff
@@ -907,39 +911,41 @@ export const InboundSAMLForm: FunctionComponent<InboundSAMLFormPropsInterface> =
                         </Grid.Row>
                         {
                             isArtifactBindingAllowed
-                            ? (
-                                <Grid.Row columns={1}>
-                                    <Grid.Column mobile={16} tablet={16} computer={16}>
-                                        <Field
-                                            ref={signatureValidationForArtifactBinding}
-                                            name="signatureValidationForArtifactBinding"
-                                            label=""
-                                            required={false}
-                                            requiredErrorMessage="this is needed"
-                                            type="checkbox"
-                                            value={
-                                                isArtifactBindingEnabled && initialValues?.singleSignOnProfile
-                                                    .enableSignatureValidationForArtifactBinding ?
-                                                    ["enableSignatureValidationForArtifactBinding"] : []}
-                                            children={[
-                                                {
-                                                    label: t("console:develop.features.applications.forms.inboundSAML" +
-                                                        ".sections.ssoProfile.fields.artifactBinding.label"),
-                                                    value: "enableSignatureValidationForArtifactBinding"
+                                ? (
+                                    <Grid.Row columns={1}>
+                                        <Grid.Column mobile={16} tablet={16} computer={16}>
+                                            <Field
+                                                ref={signatureValidationForArtifactBinding}
```

Comment:
Can we leave a space between the curly brace and the prop value? Please check other places as well.

Reply:
Addressed those props in the relevant PR.